tooling/api/src/http.ts

```diff
@@ -37,7 +37,7 @@ enum ResponseType {
   Binary = 3
 }
 
-type Part = string | number[]
+type Part = string | Uint8Array
```

Comment:
I think this causes issues at the form() function on line 60 (can't add the comment there)

Reply:
Yikes, yeah good catch. I think I'll add an internal `serializePayload` function or smth.